planet/cli/orders.py

```diff
@@ -57,18 +57,21 @@ def orders(ctx, base_url):
               help='Filter orders to given state.',
               type=click.Choice(planet.clients.orders.ORDER_STATE_SEQUENCE,
                                 case_sensitive=False))
-@click.option('-l',
-              '--limit',
-              help='Filter orders to given limit.',
+@click.option('--limit',
+              help='Maximum number of results to return. Default is 100. A '
+                   'value of 0 means no maximum.',
               default=None,
               type=int)
 @pretty
 async def list(ctx, state, limit, pretty):
-    '''List orders'''
+    '''List orders
+
+    This command outputs a sequence of the returned order descriptions.
+    If --pretty is specified, each order description is pretty-printed.
```

Comment:
The text in line 70 could go into the help string for the --pretty option?

Reply:
Yes, good call!